.clang-format

```diff
@@ -1,3 +1,5 @@
+# SPDX-FileCopyrightText: Helmholtz-Zentrum Dresden-Rossendorf e.V. <https://www.hzdr.de>
```

Comment:
but maybe `copyright` != `Urheberrecht`

Reply:
OK I think `copyright` is `Urheberrecht`. So the record you added is fine from the technical point of view but I do not think we should apply these changes. Because if we would like to change the license of the project we need the OK from the legal departments of the institutions where we currently need the OK from the contributors.